common/types/pb/pb.go

```diff
@@ -93,6 +94,18 @@ func (pbdb *Db) RegisterDescriptor(fileDesc protoreflect.FileDescriptor) (*FileD
 	if found {
 		return fd, nil
 	}
+	// Make sure to search the global registry to see if a protoreflect.FileDescriptor for
+	// the file specified has been linked into the binary. If so, use the copy of the descriptor
+	// from the global cache.
+	//
+	// Note: Proto reflection relies on descriptor values being object equal rather than object
+	// equivalence. This choice means that a FieldDescriptor generated from a FileDescriptorProto
+	// will be incompatible with the FieldDescriptor in the global registry and any message created
+	// from that global registry.
```

Comment:
Document somewhere the internal constraint that the cel library must not register anything in protoregistry.GlobalFiles as doing so might violate the isolation needed by ExprService.

Reply:
Done.